docs/general-information/data-types.md

```diff
@@ -61,9 +60,24 @@ Enumeration, may have the following values:
 
 Enumeration, may have the following values:
 
-* SUSPEND
 * ACTIVATE
+* SUSPEND
 * RESET
+* THROTTLE
+* UNTHROTTLE
+
+## NetworkSpeeds
+
+Enumeration, may have the following values:
+
+* KBIT_32
+* KBIT_64
+* KBIT_128
+* KBIT_256
+* KBIT_512
+* MBIT_1
+* MBIT_2
+* MBIT_5
```

Comment:
No full or fast speed?

Reply:
I relied on the API documentation as truth source. It is defined like so:

```java
    @ApiObjectField(
            required = false,
            allowedvalues = {"KBIT_32", "KBIT_64", "KBIT_128", "KBIT_256", "KBIT_512", "MBIT_1", "MBIT_2", "MBIT_5"}
    )
    private NetworkSpeeds networkSpeed;
```

Reply:
🤦 I momentarily forgot how we unthrottle. Carry on.